templates/squid.conf.header.erb

```diff
@@ -59,7 +59,11 @@ via                           <%= @via?'on':'off' %>
 httpd_suppress_version_string <%= @httpd_suppress_version_string?'on':'off' %>
 <% end -%>
 <% unless @forwarded_for.nil? -%>
+<% if [true, false].include? @forwarded_for -%>
```

Comment:
That's a nice way to work around the lack Boolean type. Something like `.is_a?(Boolean)` doesn't work.